code/game/gamemodes/cult/runes.dm

```diff
@@ -452,7 +452,7 @@ structure_check() searches for nearby cultist structures required for the invoca
 
 	var/input_rune_key = input(user, "Choose a rune to teleport to.", "Rune to Teleport to") as null|anything in potential_runes //we know what key they picked
 	var/obj/effect/rune/teleport/actual_selected_rune = potential_runes[input_rune_key] //what rune does that key correspond to?
-	if(!src || !Adjacent(user) || QDELETED(src) || user.incapacitated() || !actual_selected_rune)
+	if(!src || QDELETED(src) || !actual_selected_rune || QDELETED(actual_selected_rune) ||!Adjacent(user) || user.incapacitated())
```

Comment:
```suggestion
	if(QDELETED(src) || QDELETED(actual_selected_rune) || !Adjacent(user) || user.incapacitated())
```
`!src` is checked in `QDELETED` same for `actual_selected_rune`

Reply:
Oh, that's actually really good to know.